main.tf

```
@@ -38,6 +38,17 @@ locals {
       extra_config        = var.runner_install.start_script
   })
 
+  # "16.0.3" -> ["16", "0", "3"]
+  runner_version_split            = split(".", var.runner_gitlab.runner_version)
+  # ["16", "0", "3"] -> 16
+  runner_version_major            = parseint(local.runner_version_split[0], 10)
+  # ["16", "0", "3"] -> 0
+  runner_version_minor            = parseint(local.runner_version_split[1], 10)
+  # https://docs.gitlab.com/runner/executors/docker_autoscaler/#example-aws-autoscaling-for-1-job-per-instance
+  # GitLab 16.10 and earlier, manually install the plugin
+  # GitLab 16.11 and later, done through `gitlab runner fleeting install`
+  runner_use_new_fleeting_install = local.runner_version_major >= 16 && local.runner_version_minor >= 11
```

issue: seems that it does not work for `17.10` --> false, but `17.11` returns true.

```suggestion
  runner_use_new_fleeting_install = (local.runner_version_major == 16 && local.runner_version_minor >= 11) || local.runner_version_major > 16
```